metanode/partition_op_extent.go

```diff
@@ -89,6 +94,11 @@ func (mp *metaPartition) ExtentAppend(req *proto.AppendExtentKeyRequest, p *Pack
 // ExtentAppendWithCheck appends an extent with discard extents check.
 // Format: one valid extent key followed by non or several discard keys.
 func (mp *metaPartition) ExtentAppendWithCheck(req *proto.AppendExtentKeyWithCheckRequest, p *Packet) (err error) {
+	// NOTE: we cannot prevent extent append
+	// even the partition is forbidden
+	// because when we write a file,
+	// we write to new extent first, and then append them
```

Comment:
need also limit once forbidden is true, to avoid new write operation.

Reply:
If we prevent it, write operation may product orphan extent.